src/NuGet.Clients/NuGet.VisualStudio.Implementation/NullableAnnotations.cs

```diff
@@ -0,0 +1,91 @@
+// Licensed to the .NET Foundation under one or more agreements. The .NET Foundation licenses this file to you under the MIT license. See the LICENSE.md file in the project root for more information.
```

Comment:
@zkat better to match the original dotnet license text, or ours?....
```suggestion
// Copyright (c) .NET Foundation. All rights reserved.
// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
```

Reply:
This file is originally MIT, copied from dotnet/runtime. I believe other repos leave this intact.